### DISCUSSION TITLE
Testing with def_var_quantize

### DISCUSSION THREAD

**Discussion by jedwards4b** [Unidata/netcdf-c#2609] [General]
I am wondering if you have some thoughts about how to test with def_var_quantize?
Typically ParallelIO testing involves writing some simple data then reading it back and comparing
what is read to the data written - are there some analytic datasets that would allow us to do this kind
of test with the various flags and levels of nc_def_var_quantize?

**Comment by edwardhartnett** [comment]
If you check at the quantize tests in nc_test4 you will find that one way I test is to store some numbers with quantize turned on, and then check that I get the exact results I want.

I think from the PIO point of view, a good test would be to store an array of numbers, and then check that the results equal the original array, +- a small epsilon.
